### PR TITLE
Add sectors and groups in a migration

### PR DESCRIPTION
**Pull request by sleepycat**
This commit creates the data for groups and sectors using a migration.
This is needed because the existing seeding doesn't provide all the values
needed, and isn't idempotent, meaning that it can't be run as an initContainer,
which gets run every time the pod starts.

### CODE REVIEW

File: api/migrations/versions/ffa0779887d6_.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+			{id: 65, 'sector_id': 1, 's_group':'S4', 'description':'Maintain'},
+		]
+	)
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    # XXX: CASCADE nukes organisations here?
+    op.execute('truncate table groups CASCADE')
```
Comment (IdezHD) [inline api/migrations/versions/ffa0779887d6_.py]: I would use `DELETE FROM groups` and sectors keeping tables intact

**Comment by IdezHD** [comment]
Just realized this is a test breaking change, as we are creating the values per test which will conflict with data already in DB

File: api/migrations/versions/ffa0779887d6_.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+		column('s_group', String),
+		column('sector_id', Integer),
+		column('description', String)
+	)
+
+
+    # TODO: so many duplicates!
+	op.bulk_insert(groups_table,
+		[
+			{id: 1, 'sector_id': 1, 's_group':'GC_A', 'description':'Arts'}, # duplicates sector
```
Comment (IdezHD) [inline api/migrations/versions/ffa0779887d6_.py]: Check sector ID as they are all being assigned to the Sector with ID of 1

**Comment by sleepycat** [comment]
Closing in favor of a different strategy.

**Pull request closed without merging**